simulation/src/test/resources/report.md

```diff
@@ -1,24 +1,24 @@
+                                        all_nodes_500[ROUND_ROBIN].txt:	success=50.0%	client_mean=PT0.6S         	server_cpu=PT2M           	client_received=200/200	server_resps=200	codes={200=100, 500=100}
+                                   black_hole[CONCURRENCY_LIMITER].txt:	success=90.0%	client_mean=PT0.6S         	server_cpu=PT1M48S        	client_received=180/200	server_resps=180	codes={200=180}
+                                           black_hole[ROUND_ROBIN].txt:	success=65.0%	client_mean=PT0.6S         	server_cpu=PT1M18S        	client_received=130/200	server_resps=130	codes={200=130}
+                             drastic_slowdown[CONCURRENCY_LIMITER].txt:	success=100.0%	client_mean=PT0.131675S    	server_cpu=PT8M46.702333319S	client_received=4000/4000	server_resps=4000	codes={200=4000}
+                                     drastic_slowdown[ROUND_ROBIN].txt:	success=100.0%	client_mean=PT8.340639S    	server_cpu=PT9H16M2.557999978S	client_received=4000/4000	server_resps=4000	codes={200=4000}
+                        fast_500s_then_revert[CONCURRENCY_LIMITER].txt:	success=76.7%	client_mean=PT0.055281S    	server_cpu=PT3M27.306499709S	client_received=3750/3750	server_resps=3750	codes={200=2875, 500=875}
+                                fast_500s_then_revert[ROUND_ROBIN].txt:	success=76.7%	client_mean=PT0.055281S    	server_cpu=PT3M27.306499709S	client_received=3750/3750	server_resps=3750	codes={200=2875, 500=875}
+                               live_reloading[CONCURRENCY_LIMITER].txt:	success=58.3%	client_mean=PT0.7228S      	server_cpu=PT4M49.12S     	client_received=400/400	server_resps=400	codes={200=233, 500=167}
+                                       live_reloading[ROUND_ROBIN].txt:	success=58.3%	client_mean=PT0.7228S      	server_cpu=PT4M49.12S     	client_received=400/400	server_resps=400	codes={200=233, 500=167}
+             one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER].txt:	success=67.6%	client_mean=PT0.6S         	server_cpu=PT5M6S         	client_received=510/510	server_resps=510	codes={200=345, 500=165}
+                     one_endpoint_dies_on_each_server[ROUND_ROBIN].txt:	success=67.6%	client_mean=PT0.6S         	server_cpu=PT5M6S         	client_received=510/510	server_resps=510	codes={200=345, 500=165}
+                       simplest_possible_case[CONCURRENCY_LIMITER].txt:	success=100.0%	client_mean=PT0.7998S      	server_cpu=PT13M19.8S     	client_received=1000/1000	server_resps=1000	codes={200=1000}
+                               simplest_possible_case[ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.7998S      	server_cpu=PT13M19.8S     	client_received=1000/1000	server_resps=1000	codes={200=1000}
+                        slow_503s_then_revert[CONCURRENCY_LIMITER].txt:	success=100.0%	client_mean=PT0.129949S    	server_cpu=PT6M29.847333275S	client_received=3000/3000	server_resps=3130	codes={200=3000}
```

Comment:
this is an example where our RetryingChannel kicked in and resulted in servers producing responding 3130 responses, but clients just perceived 3000 green responses (as the retries were eventually successful).